src/renderers/shaders/ShaderChunk/shadowmap_vertex.glsl

```diff
@@ -2,7 +2,36 @@
 
 	for( int i = 0; i < MAX_SHADOWS; i ++ ) {
 
-		vShadowCoord[ i ] = shadowMatrix[ i ] * worldPosition;
+		#if defined(POINT_LIGHT_SHADOWS)
+
+			// if shadowDarkness[ i ] < 0.0, that means we have a point light with a cube
+			// shadow map
+			if( shadowDarkness[ i ] < 0.0 ) {
+
+				// When we have a point light, the @shadowMatrix uniform is used to store
+				// the inverse of the view matrix, so that we can get the world-space
+				// position of the light.
+
```

Comment:
Ah, I didn't know there was necessarily an orthonormal guarantee for the view matrix. I originally included the inverse of the view matrix in ```shadowMatrix ``` because I didn't want to calculate the inverse in the vertex shader. But as @tschw mentioned, since the view matrix is orthonormal we can just use the transpose as the inverse, or even better use pre-multiplication instead of post :)

@tschw You mentioned that I can use the same trick in the software cubemapper... are you saying I should avoid re-purposing vShadowCoord altogether and calculate the ray in the fragment shader? This would mean a lot of extra matrix multiplications..is that acceptable?